inc/toolbar.php

```diff
@@ -271,7 +271,7 @@ function toolbar_signature(){
     $sig = str_replace('@MAIL@',$INFO['userinfo']['mail'],$sig);
     $sig = str_replace('@DATE@',dformat(),$sig);
     $sig = str_replace('\\\\n','\\n',addslashes($sig));
-    return $sig;
+    return json_encode($sig);
```

Comment:
As far as I can see, quotes are now double-encoded due to the additional call to `addslashes` above.